app/steps/ui/documents/template.html

```diff
@@ -3,127 +3,61 @@
+    {% set checklistOneItems = (checklistOneItems.push(content["checklist2-item4-iht205"]), checklistOneItems) %}
+{% endif %}
+{% if fields.hasRenunciated.value === "true" %}
+    {% set checklistOneItems = (checklistOneItems.push(content["checklist2-item5-renunciated"] | replace('{renunciationFormLink}', links.renunciationForm)), checklistOneItems) %}
+{% endif %}
+{% if fields.executorsNameChangedByDeedPollList.value | length > 0 %}
+    {% for executor in fields.executorsNameChangedByDeedPollList.value %}
+        {% set checklistOneItems = (checklistOneItems.push(content["checklist2-item6-deed-poll"] | replace('{executorCurrentName}', executor)), checklistOneItems) %}
+    {% endfor %}
 {% endif %}
 
 {% block question %}
-    {% if fields.isDocumentUploadToggleEnabled.value === 'true' %}
-        <h1 class="form-title heading-large">{{ content.header }}</h1>
-    {% else %}
-        <h1 class="form-title heading-large">{{ content.old_header }}</h1>
-    {% endif %}
+    <h1 class="form-title heading-large">{{ content.header }}</h1>
 {% endblock %}
 
 {% block form_content %}
-    {% if fields.isDocumentUploadToggleEnabled.value === 'true' %}
-        <h2 class="heading-medium">{{ content.heading1 | safe }}</h2>
-        <p>{{ content.coverSheetDownloadMessage }} <a id="coverSheetPdfHref"
+    <h2 class="heading-medium">{{ content.heading1 | safe }}</h2>
+    <p>{{ content.coverSheetDownloadMessage }} <a id="coverSheetPdfHref"
                                                       href="/cover-sheet-pdf">{{ content.coverSheetPdf }}</a>.</p>
-        {% if fields.ccdReferenceNumber.value !== '' %}
-            <p>{{ content.text1 | safe | replace('{ccdReferenceNumber}', fields.ccdReferenceNumber.value) }}</p>
-        {% endif %}
-        <p>{{ content.text2 | safe }}</p>
-        {{ unorderedList(checklistOneItems) }}
+    {% if fields.ccdReferenceNumber.value !== '' %}
+        <p>{{ content.text1 | safe | replace('{ccdReferenceNumber}', fields.ccdReferenceNumber.value) }}</p>
+    {% endif %}
+    <p>{{ content.text2 | safe }}</p>
+    {{ unorderedList(checklistOneItems) }}
 
-        <p>{{ content.text3 | safe }}</p>
-        <p>{{ content.text4 | safe }}</p>
+    <p>{{ content.text3 | safe }}</p>
+    <p>{{ content.text4 | safe }}</p>
 
-        <div class="panel panel-border-wide">
-            <p>{{ content.warning | safe }}</p>
-        </div>
+    <div class="panel panel-border-wide">
+        <p>{{ content.warning | safe }}</p>
+    </div>
 
-        <h2 class="heading-medium">{{ content.heading2 | safe }}</h2>
+    <h2 class="heading-medium">{{ content.heading2 | safe }}</h2>
         <p>{{ content.text5 | safe }}</p>
 
-        <div class="panel panel-border-narrow">
-            <p>{{ fields.registryAddress.value | safe | nl2br }}</p>
-        </div>
-
-        <input id="sentDocuments" type="hidden" name="sentDocuments" value="true"/>
-    {% else %}
-        <p>{{ content.old_printPage | safe }}</p>
```

Comment:
Hi Megan, can you please check if the content.old_XXXX value are still being used, if not these should be removed. The content values are defined in the resources packages (app/resources/ui/translation/documents.json).

Reply:
ok resolved this be removing old document content values.